controllers/dashboard.js

```diff
@@ -187,6 +187,11 @@ router.get('/', (req, res, next) => {
 				homeworks.title = homeworks.name;
 				homeworks.private = true;
 			}
+			if (homeworks.lessonId != null) {
+				homeworks.lessonHidden = homeworks.lessonId.hidden;
+			} else {
+				homeworks.lessonHidden = false;
+			}
```

Comment:
sometimes I like to skip the else
            homeworks.lessonHidden = false;
			if (homeworks.lessonId != null) {
				homeworks.lessonHidden = homeworks.lessonId.hidden;
			}

Reply:
absolutely agree, just wanted to follow the working structure of the old code, as its allready hard to get through and this way its at least all the same way ^^